rest-api-spec/src/main/resources/rest-api-spec/test/indices.get/10_basic.yml

```diff
@@ -53,49 +53,25 @@ setup:
   - is_true: test_index.mappings
 
 ---
-"Get index infos for mappings only":
-
-  - do:
-      indices.get:
-        index: test_index
-        feature: _mapping
```

Comment:
This test was eliminated because it was for the `indices.get` API, whereas we already have tests for the `indices.get_mapping` API (in another file), so this was just a duplicate test.